ui/pages/confirmations/hooks/alerts/transactions/useResimulationAlert.ts

```diff
@@ -0,0 +1,37 @@
+import { useMemo } from 'react';
+
+import { Alert } from '../../../../../ducks/confirm-alerts/confirm-alerts';
+import { useI18nContext } from '../../../../../hooks/useI18nContext';
+import { Severity } from '../../../../../helpers/constants/design-system';
+import { RowAlertKey } from '../../../../../components/app/confirm/info/row/constants';
+import { useConfirmContext } from '../../../context/confirm';
+
+export function useResimulationAlert(): Alert[] {
+  const t = useI18nContext();
+  const { currentConfirmation } = useConfirmContext();
+
+  if (!currentConfirmation) {
+    return [];
+  }
+
+  const { simulationData } = currentConfirmation;
+  const { isReSimulatedDueToSecurity } = simulationData;
+
+  return useMemo(() => {
+    if (!isReSimulatedDueToSecurity) {
```

Comment:
Exact name of the property tbd - but this is `isReSimulatedDueToSecurity` for this preview version